packages/vue-client/src/views/RulesListView.vue

```diff
@@ -0,0 +1,45 @@
+<script setup lang="ts">
+import { toUpperCaseFirstLetter } from "@/utils/format-utils";
+import { getVariantsWithRulesDescription } from "@ogfcommunity/variants-shared";
+import { RouterLink } from "vue-router";
+
+const variantsWithRulesDescription = getVariantsWithRulesDescription();
+</script>
+
+<template>
+  <main>
+    <h1>Rule Descriptions</h1>
+    <div class="rule-links-container">
+      <RouterLink
+        v-for="variant in variantsWithRulesDescription"
+        :key="variant"
+        class="rules-link"
+        v-bind:to="{ name: 'rules', params: { variant: variant } }"
```

Comment:
FYI: Because of the inconsistency between `:key` and `v-bind:to` I want to point out, that `:` is a shorthand for `v-bind:`. See [Vue's v-bind directive](https://vuejs.org/api/built-in-directives.html#v-bind)

Reply:
I've changed it, will try to be more consistent! 😄